web/settings.py

```
@@ -203,6 +203,31 @@
+    # Validate API keys are provided when services are enabled
+    if USE_OPENAI and not OPENAI_API_KEY:
+        print("Warning: USE_OPENAI is enabled but OPENAI_API_KEY is not provided")
+    if USE_GEMINI and not GEMINI_API_KEY:
+        print("Warning: USE_GEMINI is enabled but GEMINI_API_KEY is not provided")
+    
+    try:
+        from .ai_service import AIService
+        ai_service = AIService()
+    except ImportError as e:
+        print(f"Error importing AI service: {e}")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)